app/build.gradle.kts

```diff
@@ -82,6 +87,14 @@ android {
                 }
             }
         }
+
+        unitTests.all {
+            it.testLogging {
+                events("failed", "standardOut", "standardError") // Log events
+                exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL // Show full stack traces
+                showStackTraces = true
+            }
+        }
```

Comment:
AFAIK, `unitTests` shouldn't affect instrumentation tests. So this shouldn't have any effect. Are you actually seeing this do something?